ansible/environments/local/group_vars/all

```diff
@@ -2,6 +2,7 @@
 # license agreements; and to You under the Apache License, Version 2.0.
 
 whisk_version_name: local
+extraEnvSets: "{{ lookup('env', 'OPENWHISK_EXTRA_ENV_SETS')|default(false, true) }}"
```

Comment:
We should not assume that is one top level set and force on both controller and invoker
Maybe user wants to set certain config or secret that is intended for controller but not invoker.
Or you want to set the same variable ie apikey to the db and is different for both controller and invoker.

What about having namespace with option to override
```
invoker:
   extraEnv: "{{ invoker_extra_env | lookup('env', 'OPENWHISK_EXTRA_ENV') | default(false, true) }}"

controller:
   extraEnv: "{{ controller_extra_env | lookup('env', 'OPENWHISK_EXTRA_ENV') | default(false, true) }}"
```

Reply:
Makes sense. Update PR to support specific env for controller and invoker. By default they both rely on `OPENWHISK_EXTRA_ENV`. However one can change the setting in environment specific file.

Not sure on best practices for placing the vars in ansible ... for now I have added them to `local/group_vars/all` such that one can use a different setting in specific env

```
ansible/environments/custom/group_vars
controller:
   extraEnv: "cosmos,splunk"
```